example_configs/6_pack_external_huanyang.yaml

```diff
@@ -0,0 +1,173 @@
+board: 6 Pack
+name: 6 Pack External XYZ Haunyang
+stepping:
+  engine: I2S_STREAM
+  idle_ms: 250
+  pulse_us: 2
+  dir_delay_us: 1
```

Comment:
Just spotted this. Normally dir / disable delays are higher than pulse. I'd rather just leave them out, than to specify them incorrectly.